const promise4=new Promise(function(resolve,reject){
//     setTimeout(function(){
//         let error=true;
//         if(!error){
//             resolve({name:"rohit",age:56})
//         }else{
//             reject("something went wrong")
//         }
//     },1000)
// })

// //consume the error
// promise4.catch(function(error){
//     console.log(error);
// })